docs/user-guide/deferred-download.rst

```diff
@@ -193,6 +273,24 @@ following is a basic configuration with inline documentation::
 
 For more information about a configuration option, please consult the Squid documentation.
 
+If you choose to use Varnish instead of Squid, there are two configuration files to look at.
```

Comment:
You might want to add headings for Squid and Varnish so the user can navigate more easily to the one they want to use.